docs/project-requirements.md

```diff
@@ -236,32 +236,38 @@ W każdym story pomijamy aktora, jest napisane w nagłówku kto nim jest. Dodatk
 # Kryteria akceptacji projektu dla I semestru prac
 
 ## Wymagane
-- Wybór gracza do pojedynku (US. 1.2.2)
-- Pojedynek między dwoma graczami (US. 1.1.8, 1.1.9)
-- Talia Kart (US. 1.1.13, 1.4.2 - 1.4.7, 1.1.15)
-- Profil Użytkownika (US. 1.4.1, 1.4.2, 1.1.5, 1.1.6, 1.1.14)
+- Pojedynek między dwoma graczami - symulator
 - Edytor Kart (US. 2.2.1 - 2.2.9, 2.4.4)
-- 20 Kart stworzonych w edytorze (US. 1.1.19)
-- Testy użyteczności trybu Battle i Edytora Kart
-- Serwer produkcyjny dla grupy odbiorców testującej produkt
 
 ## Oczekiwane
-- System rankingowy Battle (US. 1.1.10, 1.1.12, 1.1.18)
-- Balans kart (US. 1.1.12, 1.1.17)
-- Tabele rankingowe (US. 1.3.1 - 1.3.7)
+- Serwer produkcyjny dla grupy odbiorców testującej produkt
 - Zgłaszanie błędów / feedback (US. 1.7.2)
 
 ## Planowane
+- Testy użyteczności trybu Battle i Edytora Kart
+- 20 Kart stworzonych w edytorze (US. 1.1.19)
 - Pokrycie kodu testami jednostkowymi (Niefunkc. 4)
 - Dokumentacja deweloperska na GitHub Wiki obejmująca dotychczasowe prace (Niefunkc. 8, 9)
 - Administracja treściami zgłaszanych przez użytkowników (US. 2.2.10, 3.3, 3.4, 2.4.1, 2.4.2, 1.7.5)

```

Comment:
To można przerzucić do "Oczekiwanych"

Reply:
Na frontendzie nie będzie na razie żadnych testów jednostkowych, nie ogarnąłem ich jeszcze i nie ma raczej na to czasu, więc "Pokrycie kodu testami jednostkowymi" zostawiłbym w planowanych, bo pewnie na backendzie będą a na froncie nie

Reply:
Administracja treściami zgłaszanych przez użytkowników to nie wiem czy do Oczekiwanych czy zostawić do planowanych, wy pewnie lepiej wiecie, bo chcieliście coś terminalowego zrobić z tego co zrozumiałem

Reply:
Dokumentacja deweloperska rzeczywiście może być przeniesiona do Oczekiwanych

Reply:
Co do testów jednostkowych, mogę pokryć część kodu w weekend nimi (chyba, że wcześniej wyrobie się z wersją desktop). Natomiast nie sądzę, żeby to był taki must have na teraz. Natomiast Wirus, jak chcesz, mogę Ci w czwartek zrobić szkolenie z Jesta i możemy chociaż coś napisać ^^